src/main/scala/ai/privado/languageEngine/python/tagger/collection/CollectionTagger.scala

```diff
@@ -63,13 +63,15 @@ class CollectionTagger(cpg: Cpg, sourceRuleInfos: List[RuleInfo]) extends ForkJo
         // TODO: handle cases where `request.args.get('id', None)` used directly in handler block without method param
         val parameters =
           collectionMethod.parameter.where(_.name(sourceRule.combinedRulePattern)).whereNot(_.code("self")).l
-        val matchingLocals = collectionMethod.local.code(sourceRule.combinedRulePattern).l
+        val matchingLocals   = collectionMethod.local.code(sourceRule.combinedRulePattern).l
+        val matchingLiterals = collectionMethod.literal.code(sourceRule.combinedRulePattern).l
```

Comment:
@hiteshbedre Don't match literal's like this, it will result in a lot of FP. Refer the literalTagger in Java.
We can tag literals only when they are argument of the `get` kind of function calls

Reply:
@khemrajrathore This was really really helpful. Can able to see lots of false positive literals can be eliminated with this well handled query🙌🏻
Commit: https://github.com/Privado-Inc/privado-core/pull/379/commits/487a37602f827be6fc28a9d76c0e58afd1e33011